src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
I think we should add an ESLint configuration file. I'm used adding trailing semi-colons for each statements, but it doesn't seem to be so in that project. So every times I use my test-editor, it complains about missing semi-colons.